vhdbuilder/packer/install-dependencies.sh

```diff
@@ -106,6 +106,13 @@ net.ipv6.conf.all.forwarding = 1
 net.bridge.bridge-nf-call-iptables = 1
 EOF
 
+echo "set read ahead size to 15380 KB"
```

Comment:
this is the Ubuntu image, what about Mariner support?

Reply:
I think that both Ubuntu and Mariner will execute this script, but I'm not 100% sure.